conneg-by-ap/index.html

```diff
@@ -281,6 +281,32 @@ <h3>Archival Resource Key (ark)</h3>
 		  </p>
 	  </section>
     <div class="issue" data-number="383"></div>
+    <section id="related-altviews">
+      <h3>Linked Data APIs</h3>
+      <p>
+        Several Linked Data API tools provide access to different representations of resources, identified by URI, via
+        use of Query String Arguments (QSAs). Two examples are the
+        <a href="http://epimorphics.github.io/elda/current/index.html">ELDA tool by Epimorphics</a>, an
+        implementation of the <a href="https://github.com/UKGovLD/linked-data-api/blob/wiki/Specification.md">Linked
+        Data API</a> specification created by the government of the United Kingdom, and the <a href="">pyLDAPI</a> Python
+        Linked Data API implementation by <a href="https://www.csiro.au/">CSIRO</a>. Both ff these tools allow the
+        listing of <em>views</em> and <em>formats</em> available for a resource by dereferencing the resource URI + the
+        QSA <code>_view=alternates</code>. The <em>view</em> named <em>alternates</em> is a special view available for
+        all resources.
+      </p>
+      <p>
+        What these tools call a <em>view</em> is analogous to a profile and a <em>format</em> to an HTTP Content
+        Type.
+      </p>
+      <p>
+        These APIs provide for direct access to resource representations, conforming to a particular view, in a
+        particular format, or a default view/format if either is unspecified or invalid. For example, using pyLDAPI, the
+        available views and formats for the address <a href="http://linked.data.gov.au/dataset/gnaf/address/GAACT714845933">GAACT714845933</a>
+        can by seen by dereferencing <a href="http://linked.data.gov.au/dataset/gnaf/address/GAACT714845933?_view=alternates">http://linked.data.gov.au/dataset/gnaf/address/GAACT714845933<strong>?_view=alternates</strong></a>
+        and the "ISO19160" <em>view</em> of that Address, listed by the Alternates View, and in the RDF Turtle
+        <em>format</em>, is available at <a href="http://linked.data.gov.au/dataset/gnaf/address/GAACT714845933?_view=ISO19160&_format=text/turtle">http://linked.data.gov.au/dataset/gnaf/address/GAACT714845933<strong>?_view=ISO19160&_format=text/turtle</strong></a>.
```

Comment:
Ampersands in URLs must be written as html entities &amp;

Reply:
Done